public/index.html

```
@@ -1144,23 +1235,21 @@ <h1>PTZ Preset Control</h1>
     <div id="fill-cam-strip" aria-label="Camera status controls"></div>
     <div id="cam-tabs"><!-- rendered by JS --></div>
     <div class="header-actions">
-      <button id="live-mode-btn" class="live" title="Toggle Live/Edit mode">Live</button>
       <button id="cut-live-btn" style="display:none" title="Cut active camera to ATEM program">Cut</button>
+      <button id="live-mode-btn" class="live" title="Toggle Live/Edit mode">Live</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)